intentkit/config/config.py

```diff
@@ -83,7 +83,11 @@ def __init__(self):
         self.payment_enabled = self.load("PAYMENT_ENABLED", "false") == "true"
         # Open API for agent
         self.open_api_base_url = self.load("OPEN_API_BASE_URL", "http://localhost:8000")
-        # CDP
+        # CDP - AgentKit 0.6.0 Configuration
+        self.cdp_api_key_id = self.load("CDP_API_KEY_ID")
```

Comment:
don't add new, use exists cdp_api_key_name and cdp_api_key_private_key

only add cdp_wallet_secret